lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -201,6 +236,38 @@ describe('LMSFilePicker', () => {
     });
   });
 
+  it('shows no buttons when the request returns an empty list of files', async () => {
```

Comment:
The dialog's built-in "Cancel" button will still be shown in this case. Perhaps "shows no continue button ..." or "shows no select button ...".